agent/pkg/up9/main.go

```diff
@@ -227,6 +227,10 @@ func syncEntriesImpl(token string, model string, envPrefix string, uploadInterva
 		connection.Close()
 	}()
 
+	lastTimeSynced := time.Now().Add(time.Duration(-uploadIntervalSec) * time.Second)
```

Comment:
Can be changed to default time

```suggestion
	lastTimeSynced := time.Time{}
```

Reply:
Done with https://github.com/up9inc/mizu/commit/174abf89022c99e0c47673d1d6bdd510b00118c1